gulpfile.babel.js

```diff
@@ -9,7 +9,7 @@ const mocha = require('gulp-mocha');
 const eslint = require('gulp-eslint');
 const gulpConnect = require('gulp-connect');
 
-const tasks = ['eslint', 'sass', 'bower', 'transpile', 'serve', 'test', 'watch'];
+const tasks = ['eslint', 'sass', 'transpile', 'test', 'serve', 'watch'];
```

Comment:
why did you take out the bower task here?

Reply:
I took it out to include it as a script in package.json. This is because nodemon was starting before the files were copied and was the reason why the application was showing a blank screen when the team tried it. So i took it to package.json just to be sure that the files needed for the application to run are copied before it actually starts.